swirl/src/lib.rs

```diff
@@ -11,8 +11,8 @@ mod registry;
 mod runner;
 mod storage;
 
-pub mod errors;
 pub mod db;
+pub mod errors;
```

Comment:
Does this need to be here since it's only moving the line down?

Reply:
This was rustfmt